### PR TITLE
Initial fixes and reorganization

### PR DESCRIPTION
**Pull request by nicolaslara**
Here are some of my fixes to your branch. Most of it is cleanup so I could run it and some reorganization of the repository. I haven't had time to read the code in detail; I'll get to that later.

But this should at least make that slow loop faster and allow the code to run both on your environment and mine.

### CODE REVIEW

File: gis_exercise_2 [hunk 1]
```diff
@@ -112,7 +112,7 @@ poly
 # Read the data into memory using Pandas
 import pandas as pd
 import numpy as np
-kruger_park = pd.read_csv('some_posts.txt', sep=',')
+kruger_park = pd.read_csv('data/some_posts.txt', sep=',')
```
Comment (SGJJSR) [inline gis_exercise_2]: where is this data folder? How does github understand from where to source the data?

**Review by SGJJSR — APPROVED** [review]

File: gis_exercise_2 [hunk 2]
```diff
@@ -66,7 +66,7 @@ geo.loc[0, 'geometry'] = poly
 # # P5. Save the GeoDataFrame into a new Shapefile called 'polygon.shp'
 # # --------------------------------------------------------------------
 # # Note: you do not need to define the coordinate reference system at this time
-outfp = r"C:\\Users\\stefa\\Documents\\Fortbildung privat\\Python\\GIS\\Data\\Exercise_2_Problem_1.shp"
+outfp = r"./outputs/results.Exercise_2_Problem_1.shp"
```
Comment (SGJJSR) [inline gis_exercise_2]: yea, I have to understand how to adequately share my code... so here, you created a new directory "outputs"? Can I physically see (on github) where this directory is and what files are stored in it?

**Review by SGJJSR — APPROVED** [review]

**Review by SGJJSR — APPROVED** [review]

**Review by SGJJSR — APPROVED** [review]

**Review by SGJJSR — APPROVED** [review]

**Comment by SGJJSR** [comment]
do you see my comments, even if I approve the merge?

**Comment by SGJJSR** [comment]
what exactly have you changed at the loop?

I can't spot the difference, except that you installed a new package
'descartes'.

btw HUGE thanks!!! I know I cannot take this for granted!



Am Fr., 8. Mai 2020 um 17:44 Uhr schrieb Nicolas Lara <
notifications@github.com>:

File: gis_exercise_2 (hunk 1 above)
Comment (nicolaslara) [reply gis_exercise_2]: I created the data folder and then added `some_posts.txt` to that folder

File: gis_exercise_2 (hunk 2 above)
Comment (nicolaslara) [reply gis_exercise_2]: Yes!, but you have to be in the right branch to see it:

SGJJSR_Python_challenges_at_SGJJSR-exercise_2_py_problems

**Comment by nicolaslara** [comment]
I had forgotten to push the loop changes!! 🙈

Here they are: https://github.com/SGJJSR/Python_challenges/pull/9

**Pull request merged**